legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -327,9 +327,11 @@
         {% if business.legalType in ['SP', 'GP'] %}
             <span> | Registration #{{business.identifier}}</span>
         {% else %}
-           {% if header.name != 'incorporationApplication' %}
-             <!-- not required for incorporationApplication -->
-             <span> | Incorporation #{{business.identifier}}</span>
+            {% if header.name == 'continuationIn' %}
+               <span> | Continuation #{{business.identifier}}</span>
+            {% elif header.name != 'incorporationApplication' %}
+               <!-- not required for incorporationApplication -->
+               <span> | Incorporation #{{business.identifier}}</span>
```

Comment:
As per the design this is not changed. Please confirm with Jacqueline
![image](https://github.com/user-attachments/assets/57a406be-a362-4800-bf28-0055966a2b88)

Reply:
Jacqueline said use "Incorporation", will change back.